doc/modules/changes/20210920_ekendall

```diff
@@ -0,0 +1,3 @@
+New: Added a print statement into log file which tells the user the total wallclock time including restarts.
+<br>
+(Elodie Kendall, 2021/09/20)
```

Comment:
you might consider adding my name here as well, what do you think? :-)

Reply:
Yes of course, sorry!